site/docs/components/cookie_compliance.mdx

```diff
@@ -0,0 +1,136 @@
+---
+name: Cookie compliance
+menu: Components
+route: /components/cookie-compliance
+---
+
+import { Playground } from "docz";
+import { StatusLabel, FileInput } from "hds-react";
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Link from "../../src/components/Link";
+
+# Cookie compliance
+<StatusLabel type="alert">Draft</StatusLabel>
+
+<LargeParagraph>
+  The cookie compliance component informs users about cookie usage. This banner is shown when they visit a website or an application for the first time.
+</LargeParagraph>
+
+
+## Principles
+
+### Cookie clauses in English (EN)
+
+#### Banner text (accordion closed)
+**Headline:** [Service name] uses cookies
+
+**Bodycopy:** This website uses essential cookies to ensure site performance and for basic analytics. In addition, we use uses targeting cookies to improve the user experience, for analytics and showing targeted content. By continuing to use the site, you agree to the use of essential cookies.
+
+#### Banner text (accordion open)
+**Headline:** [Service name] uses cookies
+
+**Bodycopy:** This website uses essential cookies to ensure site performance and for basic analytics. In addition, we use uses targeting cookies to improve the user experience, for analytics and showing targeted content. By continuing to use the site, you agree to the use of essential cookies.
+
+**Subhead:** Information about the cookies used on the website
+
+**Bodycopy:** The cookies used on this website are classified according to their purpose. You can find information about each category below and allow or deny the use of cookies.
+
+**H:** Mandatory cookies
+
+**B:** The use of mandatory cookies cannot be denied. They enable the operation of the site and affect E.g. the usability of the site.
+(Checkbox + duration + information about the cookie)
+
+**H:** Optional cookies
+
+**B:** You can accept or decline optional cookies.
+(Checkbox + duration + information about the cookie)
+
+### UI components
```

Comment:
I think we can leave the "UI components" section out from the documentation page. These are quite apparent and will be used in almost every cookie consent anyway. 

If the implementation uses composition (which I do not know of), these elements will be apparent from the code examples and Storybook stories.

Reply:
Agreed